packages/language-server/src/hover.ts

```diff
@@ -28,6 +29,7 @@ export function getHoverResponse(
   if (astPosition !== undefined) {
     const ui5Node = findUI5HoverNodeAtOffset(astPosition, model);
     if (ui5Node !== undefined) {
+      track("XML_UI5_DOC_HOVER", ui5Node.kind);
```

Comment:
and here

Reply:
see answer above

Reply:
I must have not posted the answer above...
Anyhow the `track` method I am using here is a different API that accepts the **key** of the relevant event not the value. See:

```typescript
type TrackEventsKeys = keyof typeof TRACK_EVENTS;

export function track(
  eventType: TrackEventsKeys,
  ...custom_events: string[]
): void {
  try {
    getSWA().track(TRACK_EVENTS[eventType], custom_events);
  } catch (e) {
    console.error(`failed usage analytics tracking: ${e.message}`);
  }
}
```

Reply:
oh it was just marked as resolved, never mind :)